expression/integration_test.go

```diff
@@ -10382,6 +10382,17 @@ func (s *testIntegrationSuite) TestLastInsertId(c *C) {
 	tk.MustQuery("SELECT @@last_insert_id, LAST_INSERT_ID()").Check(testkit.Rows("3 3"))
 }
 
+func (s *testIntegrationSuite) TestTimestamp(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec(`use test;`)
+	tk.MustQuery("SELECT @@timestamp;")
+	tk.MustExec("set @@timestamp = 12345;")
+	tk.MustQuery("SELECT @@timestamp;").Check(testkit.Rows("12345"))
+	tk.MustExec("set @@timestamp = default;")
```

Comment:
Please also check `select now();select now();` since it should also return the static time according to the set timestamp and should not reset for the second case (seems correct, but better be clear by adding it in the test as well).